File: python/GCD.py
# To find GCD of two numbers

num1=int(input("Enter 1st Number: "))
num2=int(input("Enter 2nd Number: "))
smallest=0
if num1>num2:
    smallest=num2
else:
    smallest=num1

for i in range(1,smallest//2):
     if num1%i==0 and num2%i==0:
         gcd=i
print("GCD is: ",gcd)

File: Java/GCD.java
// To find GCD of two numbers
import java.util.*;
class GCD{
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.println("Enter 1st Number: ");
        int num1=sc.nextInt();
        System.out.println("Enter 2nd Number: ");
        int num2=sc.nextInt();
        int smallest=0;
        int gcd=0,i;
        if(num1>num2){
            smallest=num2;
        }else{
            smallest=num1;
        }
        for(i=1;i<=smallest/2;i++){
            if(num1%i==0 && num2%i==0){
                gcd=i;
            }
        }
        System.out.println("GCD is: "+gcd);
    }
}